### DISCUSSION TITLE
Not available on winget?

### DISCUSSION THREAD

**Discussion by hakanai** [wixtoolset/issues#7164] [Questions]
I thought all tools these days could be installed by winget. WiX can't. What's up with that?

There's even some intrusive prick calling themselves "IsWiX", but I can tell you now, is not WiX.

**Comment by robmen** [comment]
First, tools do not magically appear on winget. It requires effort by someone to create the winget manifests and then publish them.

That said, you will find there is (was?) an attempt to add manifests for WiX v3, but winget is lacking a feature or something blocking the effort. I don't know the details. I was not involved.

You'll also find that the sole WiX v4 bundle _was_ published to winget in -rc.1. I plan to update it in -rc.2.

And that's the "fun part" about winget. It requires additional effort for each release. One more block of work on top of all the other things that get done as part of an Open Source project.

So, please do keep that in mind when you "expect" to find a tool on winget. Unless you paid for the effort, you just got something for free.

**Comment by ChristianSauer** [comment]
I think for dev tools dotnet tools is perfectly acceptable.
